src/pages/Chat/index.css

```diff
@@ -0,0 +1,22 @@
+/* TODO: implement Firefox equivalent */
+
+/* width */
+::-webkit-scrollbar {
+    width: 7px;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+    background: transparent;
+}
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+    background: #bbb9b9;
+    border-radius: 20px;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+    background: #555;
+}
```

Comment:
Could we maybe drop this diff from this PR and handle it separately? There's already another PR that (seems) to handle Firefox correctly as well.

Reply:
Agreed. If someone already sent a better implementation I'll gladly remove this. Just sent because it was the first thing I did.